app/views/dashboards/index.html.erb

```diff
@@ -38,8 +38,8 @@
       </h2>
       <ul>
         <li>
-          <%= link_to t(".actions.new_registration"),
-                      "#{Rails.configuration.wcrs_backend_url}/registrations/start" %>
+          <!-- TODO: restore link when internal route exists https://eaflood.atlassian.net/browse/RUBY-786 -->
```

Comment:
Can we comment out the whole `div` instead? This just leaves it as a heading and nothing else.